core/src/main/java/com/datastax/oss/driver/internal/core/metadata/MetadataManager.java

```diff
@@ -106,7 +106,18 @@ private void onConfigChanged(@SuppressWarnings("unused") ConfigChangeEvent event
             || !keyspacesBefore.equals(refreshedKeyspaces)
             || (!tokenMapEnabledBefore && tokenMapEnabled))
```

Comment:
There is one method in this class:
`setSchemaEnabled`
that calls `return refreshSchema(null, false, true);` and doesn't handle error in `whenComplete`. Shouldn't it be handled? Or should the error be propagated and handled by the clients (same as for `refreshSchema`) since this is a public method?

Reply:
`setSchemaMetadataEnabled` returns the future to the caller, so we expect it to handle potential errors.